scripts/hand_test.py

```diff
@@ -0,0 +1,81 @@
+import rospy
+from sensor_msgs.msg import JointState
+from roboy_middleware_msgs.msg import MotorCommand
+import numpy as np
+from std_srvs.srv import Trigger
+from std_msgs.msg import Empty
+
+rospy.init_node("handshake")
+topic_root = '/roboy/brain/control/'
+publisher = rospy.Publisher(topic_root + "joint_targets", JointState, queue_size=1)
+msg = JointState()
+hand_msg = MotorCommand()
+hand_msg.global_id = [42,42,44,45]
+motorcmd_pub = rospy.Publisher(topic_root + "middleware/MotorCommand", MotorCommand, queue_size=1)
+rate = rospy.Rate(100)
+
+def close_hand():
+    hand_msg.setpoint = [900]*len(hand_msg.global_id)
+    motorcmd_pub.publish(hand_msg)
+
+def open_hand():
+    hand_msg.setpoint = [0]*len(hand_msg.global_id)
+    motorcmd_pub.publish(hand_msg)
+
+def move_fingers(right_hand = True, targets = [0,0,0,0]):
+    """
+    :param target: array of finger target values between 0 (open hand) and 800 (closed hand)
+    lr: "left" or "right"
+    left_ids = [38, 39, 40, 41]
+    right_ids = [42, 43, 44, 45]
+    """
+
+    if right_hand:
+        hand_msg.global_id = [42, 42, 44, 45]
+
+    else:
+        hand_msg.global_id = [38, 39, 40, 41]
+
+    hand_msg.setpoint = [i*800 for i in targets]
```

Comment:
either impose limits on the targets between 0 and 1 or dont multiply by 800 here